src/App.jsx

```diff
@@ -23,9 +22,8 @@ const App = () => {
     <Router>
       <Navigation />
       <Routes>
-        <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
-        <Route path="/categories" element={<BookList books={books} onDelete={handleDeleteBook} />} />
-        <Route path="/add-book" element={<BookForm onAddBook={handleAddBook} />} />
+        <Route path="/" />
+        <Route path="/categories" element={<BookList books={books} onDelete={handleDeleteBook} onAddBook={handleAddBook} />} />
```

Comment:
- [ ] Hi, nice one moving the Form component as suggested by the former reviewer, please the routes should just be only two, the home route which has the Form component and the categories route which just has a text, So the Booklist and Bookform components should be in the same route and the categories on another route. Please take a look at [this repo](https://github.com/AdnanOlarmmi/Bookstore) for a better idea. 

<img width="398" alt="image" src="https://github.com/redmarverick/Bookstore/assets/41163571/02b3d6af-8f14-4e82-8069-3407efdd0571">